baker/countryProfiles.tsx

```diff
@@ -40,8 +40,8 @@ const countryIndicatorGraphers = async (): Promise<GrapherInterface[]> =>
         ).map((c: any) => JSON.parse(c.config)) as GrapherInterface[]
         return graphers.filter(
             (grapher) =>
-                grapher.hasChartTab &&
-                grapher.type === "LineChart" &&
+                (grapher.hasChartTab ?? true) &&
+                (grapher.type ?? "LineChart") === "LineChart" &&
                 grapher.dimensions?.length === 1
         )
```

Comment:
Nice catch on the defaults not making it through. Changes look good! 👌 

Here's a case where extracting this statement could help with readability,  I think.

Either

```typescript
function checkShouldShowIndicator(grapher: GrapherInterface) {
  return (
    (grapher.hasChartTab ?? true) &&
    (grapher.type ?? "LineChart") === "LineChart" &&
    grapher.dimensions?.length === 1
  );
}

return graphers.filter(checkShouldShowIndicator)
```

or (my preference to nullish coalescing)

```typescript
function checkShouldShowIndicator({
  hasChartTab = true,
  type = "LineChart",
  dimensions,
}: GrapherInterface) {
  return hasChartTab && type === "LineChart" && dimensions?.length === 1;
}

return graphers.filter(checkShouldShowIndicator)
```